docs/cryptodoc/src/05_07_slh_dsa.rst

```diff
@@ -0,0 +1,444 @@
+.. _pubkey/slh_dsa:
+
+SLH-DSA
+=======
+
+Botan's implementation of the Stateless Hash-Based Digital Signature Standard
```

Comment:
```suggestion
Botan's implementation of the StateLess Hash-based Digital Signature Standard
```

Reply:
I used the same upper case as the Specification. Since it is (somewhat) a name, I think the "based" can stay upper case. The "StateLess" is a typo?